### ISSUE TITLE
How to disable autosave of a single notebook / notebooks in a folder

### ISSUE THREAD

**Issue by NumesSanguis** [jupyterlab/jupyterlab#6481]
I'm using JupyterLab in combination with JupyterHub.

# Situation
I have set-up a read-only shared folder by following: http://tljh.jupyter.org/en/latest/howto/content/share-data.html
and then changed the permission of this shared folder that other users can only read/execute, but not write.
Users can now open the notebook, execute and even change it in memory. Saving however will trigger a warning (`File Save Error for ... Permission denied`, which is intended.

The problem is that if users have made changes in memory, the notebook tries to autosave, which fails due to not having writing permission. It's very annoying for a user to close the pop-error every x seconds.

# Question
1. How do I turn-off autosave for all notebooks in a certain folder (prevent autosave in a read-only folder)
2. If 1 is not possible, how do I turn-off autosave of a single notebook?

# Possible solutions
1. In the menu --> Settings --> Autosave Documents (turn-off)
Problem: The user might forget to turn it back on and lose data, because he/she expects her notebooks to autosave.
2. Some code block that can added as the first cell in a notebook that says turn-off autosave for this notebook.
So far haven't been able to find such code.

**Comment by berkser** [comment]
Any updates on this?

**Comment by jasongrout** [comment]
No, I don't think anyone has worked on this. If anyone starts working on this, please post here to let the community and other potential collaborators know.
